test/unittest_suite.py

```diff
@@ -31,6 +31,7 @@
             modules.append('test_katello.test_plugin')
             modules.append('test_katello.test_agent.test_pulp.test_handler')
             modules.append('test_katello.test_agent.test_pulp.test_libyum')
+            modules.append('test_katello.test_agent.test_pulp.test_libdnf')
```

Comment:
Shouldn't need this since PY3 tests will be auto discoverable by the test suite!